lib/metrics.js

```diff
@@ -12,6 +12,8 @@
 const { fromJS, Set, Map, isKeyed } = require('immutable')
 const { iso } = require('./util')
 const database = require('./database')
+const config = require('../config/variables')
+const { gc: gcConfig } = config.session
```

Comment:
To me, config.session.gc might be easier to read than gcConfig

Reply:
It's more typing but ultimately readability matters more so will fix